go.sum

```diff
@@ -92,6 +98,7 @@ github.com/stretchr/testify v1.7.1/go.mod h1:6Fq8oRcR53rry900zMqJjRRixrwX3KX962/
 github.com/stretchr/testify v1.8.0/go.mod h1:yNjHg4UonilssWZ8iaSj1OCr/vHnekPRkoO+kdMU+MU=
 github.com/stretchr/testify v1.8.1/go.mod h1:w2LPCIKwWwSfY2zedu0+kehJoqGctiVI29o6fzry7u4=
 github.com/stretchr/testify v1.8.4 h1:CcVxjf3Q8PM0mHUKJCdn+eZZtm5yQwehR5yeSVQQcUk=
+github.com/stretchr/testify v1.8.4/go.mod h1:sz/lmYIOXD/1dqDmKjjqLyZ2RngseejIcXlSw2iwfAo=
```

Comment:
A little doubt why this is added cause we just have not import new go mod: )

Reply:
I just executed `go mod tidy` command 🙄, which brought the `go.sum` file to its current state:
```shell
❯ gvm use go1.21
Now using version go1.21
❯ go mod tidy
❯ git diff
<now nothing changed>
```

Reply:
Yeah you're right, I also tested that locally, seems previous pr missed it: )